card/mod/05_standard/set/type/search_type.rb

```diff
@@ -217,7 +216,7 @@ def default_search_params
     total = card.count search_params
     return '' if limit >= total # should only happen if limit exactly equals the total
 
-    @paging_path_args = { :limit => limit, :item=> inclusion_defaults[:view] }
+    @paging_path_args = { :limit => limit, :item=> inclusion_defaults(card)[:view] }
```

Comment:
Interesting.  I would think that in general when you are doing options on nested contexts (item_view, and now the more general syntax of more pipes (|) to provide options on each nested context), that you never want the value of the options to vary item to item in a list (search/pointer).  I guess it can when we access the default value and it is overridden in card sets.